ui/app/AppLayouts/Chat/panels/communities/ChatPermissionQualificationPanel.qml

```diff
@@ -0,0 +1,122 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import utils 1.0
+import StatusQ.Core 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+
+import AppLayouts.Chat.popups.community 1.0
+import AppLayouts.Chat.views.communities 1.0
+
+Item {
+    id: root
+
+    property string channelName
+    property var assetsModel
+    property var holdingsModel
```

Comment:
Why this new property needed?  If you want to use it as an internal property, that's ok, but I'm not sure about having it as a public one since the model itself will be one of the 3 we already have (`viewOnlyHoldingsModel`, `moderateHoldingsModel` or `viewAndPostHoldingsModel`). 

Not sure if makes sense to have, for example, something like this instead:

```
enum modelType {
viewOnly,
moderate, 
viewAndPost
}

function setModelType(type) {
  if (type === XX.modelType.viewOnly)
    d.holdingsModel = root.viewOnlyHoldingsModel

  if(type ===XX. modelType.moderate)
    d.holdingsModel = root.moderateHoldingsModel

  if(type === XX.modelType.viewAndPost)
    d.holdingsModel = root.veiwAndPostHoldingsModel

}
```
And by default set it as `viewAndPostHoldingsModel`.